src/main/scala/uk/gov/nationalarchives/dp/client/Client.scala

```diff
@@ -36,6 +38,9 @@ private[client] class Client[F[_], S](clientConfig: ClientConfig[F, S])(implicit
     me: MonadError[F, Throwable],
     sync: Sync[F]
 ) {
+  val underlying: CCache[String, Entry[F[String]]] =
```

Comment:
1. Why is this called "underlying"?
2. Could this be private?

Reply:
1. This is passed as an argument to `CaffeineCache` and the argument is called underlying there.
2. Yes it can, I've changed it.